src/docs/development/ui/assets-and-images.md

```diff
@@ -340,6 +340,13 @@ NSString* path = [[NSBundle mainBundle] pathForResource:key ofType:nil];
 For a more complete example, see the implementation of the
 Flutter [video_player plugin][].
 
+### Loading iOS Images in Flutter
+
+If you are implementing Flutter as
+[Add-to-app](/docs/development/add-to-app/ios/), you may have images hosted in
```

Comment:
may => might.    (In word list, search for "may":  https://developers.google.com/style/word-list )

Reply:
done